### ISSUE TITLE
is there vim wrapper? :)

### ISSUE THREAD

**Issue by yevgenko** [nvie/gitflow#101]
Hello,

I'm just curious if you know or ever heared about vim wrapper for gitflow out there?

Thanks.

**Comment by nvie** [comment]
What functionality would such a wrapper offer in your opinion?

**Comment by yevgenko** [comment]
well, maybe native functions for easy mappings and capturing the users inputs when needed and probably few internal mappings when in specific view, e.g. commenting tag, etc. just like fugitive plugin nicely wrapping the git

**Comment by ramontayag** [comment]
Yeah as of the moment, you need to type `:Git flow feature finish my_new_feature` - while that's okay there's no autocomplete, so I have to type the branch name manually. Furthermore, I think the command can be shorted to `:Gflow feature finish my_new_feature`. I don't think this is a gitflow issue, however. I just wanted to answer @nvie's question.

**Comment by alexsanford** [comment]
When I do ":Git flow release finish xxx", vim freaks out. Instead of opening up a new buffer for me to edit the tag message, I think it tries to open it right in the vim command line. It would be nice to have something which handles this properly.

**Comment by jtriley** [comment]
Perhaps this should be a feature request to @tpope for the fugitive vim plugin (https://github.com/tpope/vim-fugitive)?

**Comment by mikedfunk** [comment]
+1

**Comment by nvie** [comment]
Closing all issues as I'm archiving this repo. git-flow lives on at [git-flow-next](https://github.com/gittower/git-flow-next) if you're still using it. Thanks for reporting!
